source/_includes/common-tasks/data_disk.md

```diff
@@ -64,3 +64,41 @@ The system will prepare the data disk and immediately reboot. The reboot will ta
 Using an USB attached SSD can draw quite some power. For instance on Raspberry Pi 3 the official Raspberry Pi power supply (PSU) only provides 2.5A which can be too tight. Use a more powerful power supply if you experience issues. Alternatively use a powered USB hub. Connect the Hub to one of the USB slots of your Raspberry Pi, and connect the SSD to the Hub. In this setup the power supply of the Hub will power the attached device(s).
 
 </div>
+
+### Migrating an external data disk to another system
+
+This section shows how to move an external data disk from one system to another.
+This can be an option if the following elements apply to your use case:
+
+- You already have a functioning Home Assistant instance (system 1) that is using an external data disk.
+- You have another, new, functioning Home Assistant instance (system 2).
```

Comment:
"Functioning" sounds like as if it must be configured or something. I'd just drop that, at least for the target system.

```suggestion
- You have another, new, Home Assistant instance (system 2).
```